### DISCUSSION TITLE
blog/my-17th

### DISCUSSION THREAD

**Discussion by giscus** [jsun969/jsundotlol#4] [Announcements]
# blog/my-17th

18 岁生日在即，谨以此文记录这个平凡的 17 岁

https://jsun.lol/blog/my-17th

**Comment by li-jia-nan** [comment]
什么时候能有 jj 哥十分之一厉害

**Reply by jsun969** [reply]
什么时候能有鸡哥百分之一厉害

**Comment by wibus-wee** [comment]
什么时候能有 jj 哥十分之一厉害 😭

**Reply by jsun969** [reply]
什么时候能有wibus百分之一厉害 😭

**Comment by zdxiaoda** [comment]
什么时候能有 jj 哥百分之一厉害😭

**Comment by Hacker-C** [comment]
什么时候能有 jj 哥百分之一厉害😭

**Comment by HsuQNian** [comment]
什么时候能有 jj 哥十分之一厉害
